.travis.yml

```diff
@@ -53,11 +53,14 @@ install:
   - cp autoconf-archive-2017.09.28/m4/ax_is_release.m4 m4/
   - cp autoconf-archive-2017.09.28/m4/ax_add_fortify_source.m4 m4/
 # TPM2-TSS
-  - git clone --depth=1 -b 2.1.0 https://github.com/tpm2-software/tpm2-tss.git
+  - git clone --depth=1 -b master https://github.com/tpm2-software/tpm2-tss.git
```

Comment:
This is a placeholder. We should change to `-b 2.3.x` once rc0 is out (tomorrow I assume)

Reply:
Done, that RC came sooner than I anticipated :smile: :tada: